docs/applications/containers/beginners-guide-to-LXD-reverse-proxy/index.md

```diff
@@ -0,0 +1,668 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'LXD is a container hypervisor that manages Linux Containers. Compared to other uses of Linux Containers, LXD manages system containers which each work just like typical servers. This guide shows how to setup a reverse proxy in LXD 3 so that it is possible to host many websites in LXD system containers.'
+keywords: ["container", "lxd", "lxc", "apache", "nginx", "reverse proxy", "virtual machine", "virtualization"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2019-08-13
+modified: 2019-08-13
+modified_by:
+  name: Linode
+title: "A Beginner's Guide to LXD: Setting Up a Reverse Proxy to Host Many Websites in Containers"
+contributor:
+  name: Simos Xenitellis
+  link: https://blog.simos.info/
+external_resources:
+  - '[LXD Introduction](https://linuxcontainers.org/lxd/)'
+  - '[LXD support community](https://discuss.linuxcontainers.org/)'
+  - '[Try LXD Online](https://linuxcontainers.org/lxd/try-it/)'
+  - '[NGINX Web Server](https://docs.nginx.com/nginx/admin-guide/web-server/)'
+  - '[Apache Web Server](https://httpd.apache.org)'
+  - '[NGINX Reverse Proxy Settings](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/)'
+  - '[Proxy Protocol](https://www.haproxy.com/blog/haproxy/proxy-protocol/)'
+  - '[TLS Termination Proxy](https://en.wikipedia.org/wiki/TLS_termination_proxy)'
+---
+
+## Introduction
+
+[LXD](https://linuxcontainers.org/lxd/) (pronounced "Lex-Dee") is a system container manager build on top of Linux Containers (LXC) and is supported by [Canonical](https://canonical.com). The goal of LXD is to provide an experience similar to a virtual machine but through containerization rather than hardware virtualization. Compared to Docker for delivering applications, LXD offers nearly full operating-system functionality with additional features such as snapshots, live migrations, and storage management.
+
+In this guide you will:
+
```

Comment:
I shortened this section into a bulleted list and made it into this new format that we started using "In this guide you will:" with links to the various sections.